Gruntfile.js

```diff
@@ -170,6 +170,8 @@ module.exports = function (grunt) {
         exec: {
             copyjs: `cp shared/js/*.js build/${browser}/${buildType}/js/ && rm build/${browser}/${buildType}/js/*.es6.js`,
             copyContentScripts: `cp shared/js/content-scripts/*.js build/${browser}/${buildType}/public/js/content-scripts/`,
+            // replace `/* __ */ 'https://duckduckgo.com' /* __ */` in content-scripts/onboarding.js for local dev
+            devifyOnboarding: `sed -i${process.platform === 'darwin' ? " ''" : "''"} "s/\\/\\* __ \\*\\/ 'https:\\/\\/duckduckgo\\.com' \\/\\* __ \\*\\//'*'/" build/${browser}/${buildType}/public/js/content-scripts/onboarding.js`,
```

Comment:
apologies for the horror here. The double escaping for the special charcaters is needed as we are in a JS template string and for what i can tell, the `sed` that comes standard with OSX (BSD?) needs `sed -i ''` (the suffix MUST be specified) whereas GNU sed is fine with just `sed -i`.

Reply:
going to check quickly if we can find a version of the command that avoids the process.platform check...